package.json

```diff
@@ -15,7 +15,20 @@
     "biojs"
   ],
   "main": "build/pileup.js",
-  "browser": "build/pileup.js",
+  "browser": "src/pileup.js",
+  "browserify": {
```

Comment:
Sounds good. If there's some way to configure wzrd.in that would be good, too. I've been pretty happy using `grunt` as our build system—I'd like to avoid having two separate systems if it's possible.